rpcs3/rpcs3qt/tooltips.h

```diff
@@ -103,6 +103,7 @@ class Tooltips : public QObject
 		const QString debug_overlay                = tr("Provides a graphical overlay of various debugging information.\nIf unsure, don't use this option.");
 		const QString log_shader_programs          = tr("Dump game shaders to file. Only useful to developers.\nIf unsure, don't use this option.");
 		const QString disable_occlusion_queries    = tr("Disables running occlusion queries. Minor to moderate performance boost.\nMight introduce issues with broken occlusion e.g missing geometry and extreme pop-in.");
+		const QString disable_video_output         = tr("Literally disables all video output and PS3 graphical rendering.\nIts only use case is to evaluate performance on CELL for development.");
```

Comment:
I think the "Literally" i unnecessary

Reply:
Also "Its" -> "It's"!

Reply:
It is correct.

Reply:
"It's" = "It is" and "It is only use case" makes no sense. "Its" is correct.